.github/workflows/e2e-csv-tempo-verification.yml

```
@@ -83,6 +83,8 @@ jobs:
             alpine:3.17
 
       - name: Build csv-payments modular observability images
+        env:
+          MAVEN_REPOSITORY: ${{ runner.home }}/.m2/repository
```

actionlint (1.7.12)

[error] property "home" is not defined in object type {arch: string; debug: string; environment: string; name: string; os: string; temp: string; tool_cache: string}

(expression)